src/components/reports/attendee-report.js

```
@@ -14,7 +14,7 @@
 import React from "react";
 const Query = require("graphql-query-builder");
```

ESLint

[error] Expected 1 empty line after require statement not followed by another require.

(import/newline-after-import)